bokehjs/src/lib/core/property_mixins.ts

```diff
@@ -250,11 +251,11 @@ export type TitleText = Prefixed<"title", Text>
 type Mixins = Text | Line | Fill | Hatch
 
 export function attrs_of<P extends string, T extends Mixins>(
-    model: any, prefix: P, mixin: p.DefineOf<T>, prefixed: boolean = false): {[key: string]: any} {
+    model: HasProps, prefix: P, mixin: p.DefineOf<T>, prefixed: boolean = false): {[key: string]: unknown} {
   const attrs: {[key: string]: unknown} = {}
   for (const attr of keys(mixin)) {
     const prefixed_attr = `${prefix}${attr}` as const
-    const value = model[prefixed_attr]
+    const value = (model as any)[prefixed_attr]
```

Comment:
@mattpap would `model.property(prefixed_attr)` help avoid `any`?

Reply:
It wouldn't help much, because `HasProps.property()` accepts a `string` name (instead of `keyof ...`), so it's as much dynamically typed as this. What has to be done, is `keys()` has to be typed with `(keyof ...)[]` instead of `string[]`. In fact, I prefer `as  any`  in such cases,  because it's very visible and has higher chance to get resolved.